src/js/controller/controller.js

```diff
@@ -428,7 +438,20 @@ Object.assign(Controller.prototype, {
 
         function _autoStart() {
             const state = _model.get('state');
-            if (state === STATE_IDLE || state === STATE_PAUSED) {
+            if (!checkAutoStartCancelable.cancelled() && (state === STATE_IDLE || state === STATE_PAUSED)) {
+                const mode = _model.get('canAutoplay');
+
+                if (mode === AUTOPLAY_DISABLED) {
```

Comment:
It's odd that we duplicate this functionality here (Re: my comment above, before I say this I thought we weren't sending it).